templates/account/email/email_confirmation_message.txt

```diff
@@ -1,8 +1,11 @@
-{% load account %}{% user_display user as user_display %}{% load i18n %}{% autoescape off %}{% blocktrans with site_name=current_site.name site_domain=current_site.domain %}
+{% load account %}
```

Comment:
Please put all these instructions on a single line, as they were before, with no newlines in between. Newlines are visible to end users, and this merge should not change the text we email users.